database/knex.js

```diff
@@ -1,11 +1,19 @@
 const expect = require("expect-runtime");
 const connection = process.env.DATABASE_URL
 expect(connection).to.match(/^postgresql:\//);
-
+const urlregexp = /postgresql:\/\/(.+):(.+)@(.+):(\d+)\/(.+)\?ssl=true/g;
+const dbConnValues = [...connection.matchAll(urlregexp)][0];
 let knexConfig = {
   client: "pg",
   debug: process.env.NODE_LOG_LEVEL === "debug" ? true : false,
-  connection: connection,
+  connection: {
+    host: dbConnValues[3],
+    user: dbConnValues[1],
+    password: dbConnValues[2],
+    database: dbConnValues[5],
+    port: dbConnValues[4],
+    ssl: { rejectUnauthorized: false }
```

Comment:
The ability to specify the ssl here is what made me choose the parsing the DATABASE_URL into `dbConnValues` to be used in the connection object here. This helps us to use node 16.3.0 successfully. We could use this as a solution for our node upgrade projects